docs/concepts/models/python_models.md

```diff
@@ -143,6 +143,28 @@ def execute(
     context.table("docs_example.another_dependency")
 ```
 
+
+## Variables
+
+[User-defined variables](../../reference/configuration.md#variables) variables can be accessed from within the Python model using the `context.var` method. For example:
+
+```python linenums="1"
+@model(
+    "my_model.name",
+)
+def execute(
+    context: ExecutionContext,
+    start: datetime,
+    end: datetime,
+    execution_time: datetime,
+    **kwargs: t.Any,
```

Comment:
the purpose of kwargs was to handle vars

Reply:
It doesn't work though cause it'd impossible to statically detect whether variable is used or not.

Reply:
I think we can do this if check which variables user specified explicitly in the list of arguments. However, I can see several issues with that:

1. A user might expect variables to be available through `kwargs` by key, but that won't be the case.
2. A user has to make sure to provide a default value for the argument, otherwise it'd fail at runtime when a variable is missing.
3. The interface looks quite different from what we do in macros.

So there's more implicit magic involved imho. Wdyt?